src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -507,6 +507,23 @@ std::string LevelZeroCompilerInDriver<TableExtension>::serializeConfig(
         content = std::regex_replace(content, std::regex(batchstr.str()), "");
     }
 
+    // DRIVER_ELF_FROMAT_VERSION and DRIVER_MI_VERSION are not supported in versions < 5.7 - need to remove them
+    if ((compilerVersion.major < 5) || (compilerVersion.major == 5 && compilerVersion.minor < 7)) {
```

Comment:
5.7 is the current version. Shouldn't we use 5.8 as the threshold?

Reply:
Fixed

Reply:
5.9 now